### ISSUE TITLE
RepeatModeler is finished, but no consensi.fa files found

### ISSUE THREAD

**Issue by minhasbushra** [oushujun/EDTA#350]
I am running EDTA with this command, but it is unable to get consensi.fa. 

_perl ${edta}/EDTA.pl --genome ${genome}/cesox.v3.fa.mod --sensitive 1 --anno 1 --evaluate 1 --threads 20 --repeatmasker /***/RepeatMasker --repeatmodeler /***/RepeatModeler-2.0.4_

Below is what the log says... Please let me know how can i fix that
PS: I don't see any directory starting with RM in my EDTA output folder

_Wed Apr 12 14:03:27 CDT 2023    Perform EDTA final steps to generate a non-redundant comprehensive TE library:

                                Use RepeatModeler to identify any remaining TEs that are missed by structure-based methods.

cat: 'RM_*/consensi.fa': No such file or directory
                                RepeatModeler is finished, but no consensi.fa files found.

                                Skipping the CDS cleaning step (--cds [File]) since no CDS file is provided or it's empty._

**Comment by minhasbushra** [comment]
Also from my understanding the database that we are using for repeat modeler is cesox.v3.fa.mod.mod.LTR.TIR.Helitron.fa.stg1 and not the raw sequence file we provided. So are the results going to be different if i have repeatmodeler out put from raw sequences? is it ok if i just combine final library generated by EDTA and combine it with the library generated by Repeat modeler ?

**Comment by oushujun** [comment]
In EDTA, RepeatModeler will run on the genome, not `stg1` libraries. Please check the *final folder for information about the repeatmodeler status. It's not recommended to directly combine the EDTA library and the repeatmodeler library since there are redundancies between them, and the repeatmodeler library is of lower quality based on our benchmarks in the paper.

Shujun

**Comment by lvqiang0120** [comment]
Thanks for your response. EDTA completed the annotation successfully. The total proportion of repeat sequence was 47.59%.  I think it was probably lower than it really was. The proportion of repetitive sequences might be around 60% based on previous results and closely related species. Can you give me some suggestions to adjust? I always had a question, EDTA pipeline had  integrated RepeatModeler and RepeatMasker, why some users still use RepeatModeler to annotate non-LTR. Although it has been suggested that EDTA may not be very accurate in its annotation of non-LTR.

**Comment by oushujun** [comment]
You may want to use a generic repeat annotator to analyze the
repetitiveness of your genome to gain a fair idea of the repetitiveness. If
your genome has lots of non-LTRs, I suggest you collect some non-LTR
sequences of this species or its sister species and give them to EDTA as a
curated library. EDTA uses RepeatModeler to identify non-LTRs but will
perform extra steps to filter. Neither EDTA nor RepeatModeler has a
sensitive annotation of non-LTRs. Please read the wiki for more details.

Shujun

**Comment by minhasbushra** [comment]
Thanks, I tried it again... I get the error message 

 ...Use RepeatModeler to identify any remaining TEs that are missed by structure-based methods.

cat: 'RM_*/consensi.fa': No such file or directory
                                RepeatModeler is finished, but no consensi.fa files found.

apparently, it is looking for the consensi file in some RM* directory (that i don't see), i saw the .consensi file in final folder but that is empty too. 

could it be due to the new repeat modeler version I m using? I have posted the command I am using earlier. Not sure how i get to run repeat modeler with EDTA. Any suggestions? Thanks
